BusBoard.Api/IncomingBusesApi.cs

```diff
@@ -0,0 +1,195 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using Newtonsoft.Json.Linq;
+
+namespace BusBoard.Api
+{
+    public class IncomingBusesApi
+    {
+        private const string PrimaryKey = "1056f6a94b6e44b48368f54a155429a3";
+
+        private static string GetJsonResponse(string url)
+        {
+            var request = (HttpWebRequest)WebRequest.Create(url);
+            var responseJson = string.Empty;
+
+            try
+            {
+                using (var response = (HttpWebResponse)request.GetResponse())
+                using (var stream = response.GetResponseStream())
+                using (var reader = new StreamReader(stream))
+                {
+                    responseJson = reader.ReadToEnd();
+                }
+
+                return responseJson;
+            }
+            catch (Exception e)
+            {
+                return null;

```

Comment:
You should probably do something with this exception (e.g. logging), but if you don't catch the exception I think you'll automatically get redirected to the error page?

Reply:
That's right